docs/quickstart-aks.md

```diff
@@ -121,6 +132,31 @@ Congratulations! You can now deploy your applications on the Kubernetes cluster
 
 - [Installing Kubeapps on BKPR](kubeapps-on-bkpr.md)
 
+## Upgrading
+
+To upgrade BKPR, first execute the following command to query the version of the latest stable release.
+
+```bash
+curl --silent "https://api.github.com/repos/bitnami/kube-prod-runtime/releases/latest" | jq -r '.tag_name'
+```
+
+Next, edit the `kubeprod-manifest.jsonnet` file that was generated by `kubeprod install` and update the version referred in the `import` statement. For example, the following snippet illustrates the changes required in the `kubeprod-manifest.jsonnet` file if you're upgrading to version `v1.1.0` from version `v1.0.0`.
+
+```diff
+ // Cluster-specific configuration
+-(import "https://releases.kubeprod.io/files/v1.0.0/manifests/platforms/aks.jsonnet") {
++(import "https://releases.kubeprod.io/files/v1.1.0/manifests/platforms/aks.jsonnet") {
+  config:: import "kubeprod-autogen.json",
+  // Place your overrides here
+ }
+```
+
+Finally, apply the update using the following command:
+
+```bash
+kubecfg update kubeprod-manifest.jsonnet --gc-tag kube_prod_runtime
```

Comment:
The "quickstart" version of upgrade looks like this:

1. Download new `kubeprod` release
2. Make above jsonnet change (I would very much like to remove this step somehow)
3. Re-run `kubeprod install aks ...` command in dir with existing `kubeprod-autogen.json` and (updated) `kubeprod-manifest.jsonnet` file

In particular, the new `kubeprod install` might do some extra non-k8s setup (and rewrite `kubeprod-autogen.json`) as part of the upgrade.  Running `kubecfg` directly won't do that.